data/json/recipes/practice/melee.json

```
@@ -149,144 +149,929 @@
+  },
+  {
+    "id": "prac_stabbing_int_knife",
+    "type": "practice",
+    "activity_level": "BRISK_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_MELEE",
+    "name": "knives (intermediate)",
+    "description": "Get in touch with your inner commando and practice some advanced knife-fighting techniques on a training dummy.",
+    "skill_used": "stabbing",
+    "skills_required": [ [ "melee", 1 ] ],
+    "time": "1 h",
+    "practice_data": { "min_difficulty": 2, "max_difficulty": 3, "skill_limit": 4 },
+    "autolearn": [ [ "stabbing", 4 ] ],
+    "book_learn": [ [ "manual_cutting", 3 ], [ "manual_knives", 2 ] ],
+    "proficiencies": [
+      { "proficiency": "prof_knives_familiar", "required": true },
+      { "proficiency": "prof_knives_pro", "time_multiplier": 2, "skill_penalty": 0 }
+    ],
+    "tools": [
+      [ "makeshift_knife", "primitive_knife", "knife_trench", "knife_combat", "knife_chef", "knife_rambo" ],
+      [ "pseudo_training_dummy_light", "pseudo_training_dummy_heavy" ]
+    ]
+  },
+  {
+    "id": "prac_stabbing_beg_shiv",
+    "type": "practice",
+    "activity_level": "BRISK_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_MELEE",
+    "name": "shivs (beginner)",
+    "description": "You wouldn't think it, but there is more to stabbing someone with something small and point.  Practice shanking an imaginary enemy where it hurts most.",
+    "skill_used": "stabbing",
+    "skills_required": [ [ "melee", 1 ] ],
+    "time": "1 h",
+    "practice_data": { "min_difficulty": 0, "max_difficulty": 1, "skill_limit": 2 },
+    "autolearn": [ [ "stabbing", 2 ] ],
+    "book_learn": [ [ "manual_knives", 1 ], [ "manual_melee", 1 ] ],
+    "proficiencies": [ { "proficiency": "prof_shivs_familiar", "time_multiplier": 1.5, "skill_penalty": 0 } ],
+    "tools": [ [ "screwdriver", "glass_shiv", "sharp_toothbrush", "scissors_child", "scissors_medical", "scissors" ] ]
+  },
+  {
+    "id": "prac_stabbing_int_shiv",
+    "type": "practice",
+    "activity_level": "BRISK_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_MELEE",
+    "name": "shivs (intermediate)",
+    "description": "Practice shanking the ever living hell out of a dummy to hone your accuracy and grip while you gut someone.",
+    "skill_used": "stabbing",
+    "skills_required": [ [ "melee", 1 ] ],
+    "time": "1 h",
+    "practice_data": { "min_difficulty": 2, "max_difficulty": 3, "skill_limit": 4 },
+    "autolearn": [ [ "stabbing", 4 ] ],
+    "book_learn": [ [ "manual_knives", 3 ], [ "manual_melee", 3 ] ],
+    "proficiencies": [
+      { "proficiency": "prof_shivs_familiar", "required": true },
+      { "proficiency": "prof_shivs_pro", "time_multiplier": 2, "skill_penalty": 0 }
+    ],
+    "tools": [
+      [ "screwdriver", "glass_shiv", "sharp_toothbrush", "scissors_child", "scissors_medical", "scissors" ],
```

```suggestion
      [ "screwdriver", "sharp_toothbrush", "scissors_child", "scissors_medical", "scissors" ],
```
Glass shivs too fragile to repeatedly stab wood